docs/Guide/preconditions/what-are-preconditions.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This page feels.. small :megu:

Which is fine
But thought I'd mention it anyways